tests/suites/cli/active_branch.sh

```diff
@@ -0,0 +1,88 @@
+# Checks juju indicates the correct branch output tabular and json.
+run_indicate_active_branch_no_active() {
+  echo
+
+  file="${TEST_DIR}/indicate-active-branch-no-active.txt"
+
+  ensure "indicate-active-branch-no-active" "${file}"
+
+  export JUJU_DEV_FEATURE_FLAGS=generations
+  juju deploy ubuntu 2>&1
```

Comment:
I'm unsure how I feel about this. What does redirecting stderr to stdout give us?

Reply:
actually not that important. Was thinking maybe stderr information could help. Lemme remove it :+1: